modules/access-cluster.adoc

```diff
@@ -7,7 +7,7 @@
 [id="access-cluster_{context}"]
 = Accessing your cluster
 
-After you have added a user to your configured identity provider, you can log in to your {product-title} cluster through the web console.
+After you have configured your identity providers, users can access the cluster from {cluster-manager-first}.
 
 .Prerequisites
 
```

Comment:
There's an {OCM} on line 14 that is showing in plaintext in preview

Reply:
This is still showing {OCM} and I'm not sure why